[2022-02-02 18:02:29,943] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2022-02-02 18:02:29,955] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2022-02-02 18:02:29,956] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 18:02:29,957] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-02 18:02:29,959] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 18:02:29,985] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-02-01 00:00:00+00:00
[2022-02-02 18:02:29,993] {standard_task_runner.py:52} INFO - Started process 8090 to run task
[2022-02-02 18:02:29,997] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-02-01T00:00:00+00:00', '--job-id', '307', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmp5t8wxkx_', '--error-file', '/tmp/tmpgislurnp']
[2022-02-02 18:02:29,999] {standard_task_runner.py:77} INFO - Job 307: Subtask download_dataset_task
[2022-02-02 18:02:30,067] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-02 18:02:30,105] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-02 18:02:30,127] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T00:00:00+00:00
[2022-02-02 18:02:30,128] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-02 18:02:30,130] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-02.csv > /opt/***/yellow_tripdata_2019-02.csv']
[2022-02-02 18:02:30,145] {subprocess.py:85} INFO - Output:
[2022-02-02 18:07:24,754] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-02 18:07:24,821] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190201T000000, start_date=20220202T180229, end_date=20220202T180724
[2022-02-02 18:07:24,893] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 18:07:24,944] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 12:47:29,548] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2022-02-10 12:47:29,563] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2022-02-10 12:47:29,564] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 12:47:29,565] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 12:47:29,565] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 12:47:29,579] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-02-01 00:00:00+00:00
[2022-02-10 12:47:29,586] {standard_task_runner.py:52} INFO - Started process 655 to run task
[2022-02-10 12:47:29,591] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-02-01T00:00:00+00:00', '--job-id', '663', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpeopaaftq', '--error-file', '/tmp/tmptj0zuh_8']
[2022-02-10 12:47:29,593] {standard_task_runner.py:77} INFO - Job 663: Subtask download_dataset_task
[2022-02-10 12:47:29,671] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [running]> on host e30f201fae61
[2022-02-10 12:47:29,721] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 12:47:29,752] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T00:00:00+00:00
[2022-02-10 12:47:29,754] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 12:47:29,755] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-02.csv > /opt/***/yellow_tripdata_2019-02.csv']
[2022-02-10 12:47:29,769] {subprocess.py:85} INFO - Output:
[2022-02-10 12:48:34,859] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 12:48:34,888] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190201T000000, start_date=20220210T124729, end_date=20220210T124834
[2022-02-10 12:48:34,927] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 12:48:34,971] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 15:29:55,232] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2022-02-10 15:29:55,262] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2022-02-10 15:29:55,264] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:29:55,265] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 15:29:55,266] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:29:55,294] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-02-01 00:00:00+00:00
[2022-02-10 15:29:55,308] {standard_task_runner.py:52} INFO - Started process 116 to run task
[2022-02-10 15:29:55,314] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-02-01T00:00:00+00:00', '--job-id', '781', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpts2j_24t', '--error-file', '/tmp/tmp_x6mt38a']
[2022-02-10 15:29:55,316] {standard_task_runner.py:77} INFO - Job 781: Subtask download_dataset_task
[2022-02-10 15:29:55,443] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [running]> on host 276a0ad90304
[2022-02-10 15:29:55,516] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 15:29:55,560] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T00:00:00+00:00
[2022-02-10 15:29:55,564] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 15:29:55,566] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-02.csv > /opt/***/yellow_tripdata_2019-02.csv']
[2022-02-10 15:29:55,590] {subprocess.py:85} INFO - Output:
[2022-02-10 15:30:47,001] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 15:30:47,092] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190201T000000, start_date=20220210T152955, end_date=20220210T153047
[2022-02-10 15:30:47,199] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 15:30:47,362] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 16:30:05,655] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2022-02-10 16:30:05,687] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [queued]>
[2022-02-10 16:30:05,689] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:30:05,690] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 16:30:05,691] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:30:05,713] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-02-01 00:00:00+00:00
[2022-02-10 16:30:05,726] {standard_task_runner.py:52} INFO - Started process 177 to run task
[2022-02-10 16:30:05,750] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-02-01T00:00:00+00:00', '--job-id', '928', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmp0y27oefg', '--error-file', '/tmp/tmpg9h0f5u4']
[2022-02-10 16:30:05,753] {standard_task_runner.py:77} INFO - Job 928: Subtask download_dataset_task
[2022-02-10 16:30:05,886] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-02-01T00:00:00+00:00 [running]> on host 106a34111faf
[2022-02-10 16:30:05,952] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 16:30:06,011] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-02-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-01T00:00:00+00:00
[2022-02-10 16:30:06,015] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 16:30:06,018] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-02.csv > /opt/***/yellow_tripdata_2019-02.csv']
[2022-02-10 16:30:06,040] {subprocess.py:85} INFO - Output:
[2022-02-10 16:31:15,250] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 16:31:15,347] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190201T000000, start_date=20220210T163005, end_date=20220210T163115
[2022-02-10 16:31:15,450] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 16:31:15,545] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
